pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto

```diff
@@ -586,6 +586,12 @@ message LinuxContainerSecurityContext {
     // no_new_privs defines if the flag for no_new_privs should be set on the
     // container.
     bool no_new_privs = 11;
+    // masked_paths is a slice of paths that should be masked by the container
+    // runtime, this can be passed directly to the OCI spec.
+    repeated string masked_paths = 13;
```

Comment:
nit: 13 -> 12?

Reply:
y, i scratched my head for a few mins and realized that

Reply:
is there any guidance for runtimes on how to handle these two fields? https://github.com/containerd/cri/commit/3e4cec8739bbb0db0764f6db80a25a3f75411c0b#diff-c656bacd6cb0fe9a7f64814b01256decR358 cri-containerd seems to handle it that way but I believe there must be guidance in the api itself, @Random-Liu @jessfraz